sagetasks/synapse.py

```diff
@@ -33,9 +33,11 @@ def run(self, auth_token, synapse_id, sep=None):
         file = self.synapse.get(synapse_id, downloadFile=False)
         file_handle_id = file._file_handle.id
         file_handle_url = self.synapse.restGET(
-            f"/fileHandle/{file_handle_id}/url",
+            f"/file/{file_handle_id}",
             self.synapse.fileHandleEndpoint,
-            params={"redirect": False}
+            params={"redirect": False,
+                    "fileAssociateType": "FileEntity",
```

Comment:
Heres the types: https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/file/FileHandleAssociation.html